packages/sku/package.json

```diff
@@ -87,6 +88,7 @@
     "browserslist": "^4.16.1",
     "browserslist-config-seek": "3.2.0",
     "chalk": "^4.1.0",
+    "commander": "^12.1.0",
```

Comment:
Oh, I forgot that we used it for the server entry too. Maybe we could move to node 20 as part of the breaking change then just use https://nodejs.org/api/util.html#utilparseargsconfig for the server entry since it's so simple.